src/ert/shared/plugins/plugin_manager.py

```diff
@@ -80,6 +81,16 @@ def __str__(self) -> str:
     def get_help_links(self) -> Dict[str, Any]:
         return ErtPluginManager._merge_dicts(self.hook.help_links())
 
+    @property
+    def forward_model_steps(
+        self,
+    ) -> List[Type[ForwardModelStep]]:
+        fm_steps_listed = [
+            resp.data for resp in self.hook.installable_forward_model_steps()
+        ]
+        all_fm_steps = [fm_step for fm_steps in fm_steps_listed for fm_step in fm_steps]
```

Comment:
```
        return [fm_step.data for fm_steps in self.hook.installable_forward_model_steps() for fm_step in fm_steps]
```

Reply:
Can rewrite into something closer to a one-liner, suggestion _ almost_ works but needs this tweak:
```
return [
            fm_step
            for fm_steps in [
                x.data for x in self.hook.installable_forward_model_steps()
            ]
            for fm_step in fm_steps
        ]
``` 
as `data` is fetched out of every entry in `self.hook.installable_forward_model_steps()` which wraps the fm step class in a `PluginResponse`